taxonomies/ingredients.txt

```diff
@@ -3243,7 +3243,7 @@ wikipedia:en:https://en.wikipedia.org/wiki/Beeswax
 # usage:fr:agent antiadhérant:cire d'abeilles
 
 <en:beeswax
-en:white and yellow beeswax
+en:white and beeswax
```

Comment:
that change seems incorrect, why remove "yellow"?

Reply:
no idea why that changed